Casks/q/qfinder-pro.rb

```diff
@@ -1,15 +1,24 @@
 cask "qfinder-pro" do
-  version "7.11.1.0726"
+  version "7.11.1,0726"
   sha256 "dffb9a2a58663d8be480eb04d440571842c0e68c5fe8ad610922725f1abc48c6"
 
-  url "https://download.qnap.com/Storage/Utility/QNAPQfinderProMac-#{version}.dmg"
+  url "https://download.qnap.com/Storage/Utility/QNAPQfinderProMac-#{version.csv.first}.#{version.csv.second}.dmg"
   name "Qnap Qfinder Pro"
   desc "NAS management application"
   homepage "https://www.qnap.com/en/utilities#utliity_5"
 
   livecheck do
     url "https://update.qnap.com/SoftwareRelease.xml"
-    regex(/QNAPQfinderProMac[._-]v?(\d+(?:\.\d+)+)\.dmg/i)
+    strategy :xml do |xml|
+      item = xml.elements["//application[productName[text()='Qfinder']]/platform[platformName[text()='Mac_for_QT']]/software"]
```

Comment:
```suggestion
      item = xml.elements[
        "//application[productName[text()='Qfinder']]" \
        "/platform[platformName[text()='Mac_for_QT']]/software"
      ]
```

Maybe not the prettiest approach but this is one way to address the "Line is too long" style issue.

Reply:
Thanks, your suggestion is less verbose than what I tried.